packages/core/faststore.config.default.js

```diff
@@ -99,4 +99,16 @@ module.exports = {
     enableCypressExtension: false,
     noRobots: false,
   },
+  rewrites: async function rewrites() {
+    return {
+      beforeFiles: [
+        {
+          source: '/test/my-office',
+          destination: `/office`,
+        },
+      ],
+      afterFiles: [],
+      fallback: [],
+    }
+  },
```

Comment:
🎗️ temporary rewrites added to this PR just for test purposes. Remember to remove before merging.